problems/arraysWithLoops.js

```diff
@@ -24,40 +40,88 @@ function shoutWhileLoop() {}
  * @param {Array} - ["I", "am", "a", "happy", "array"]
  * @returns {Array} - ["I!", "am!", "a!", "happy!", "array!"]
  */
+    // const iterable = [10, 20, 30];
+
+// for (let value of iterable) {
+//   value += 1;
+//   console.log(value);
 
-function shoutForOfLoop() {}
+  const shoutForOfLoop = (array) => {
+    let newArray = []
+      for (let i of array) {
```

Comment:
use el instead of i for for of loops because i is usually the index, but here it's the element